saleor/graphql/app/mutations/app_reenable_sync_webhooks.py

```diff
@@ -0,0 +1,72 @@
+import logging
+from typing import cast
+
+import graphene
+from django.core.exceptions import ValidationError
+
+from ....app import models as app_models
+from ....app.error_codes import AppErrorCode
+from ....graphql.app.types import App
+from ....graphql.core.descriptions import ADDED_IN_321
+from ....permission.enums import AppPermission
+from ....webhook.circuit_breaker.breaker_board import (
+    initialize_breaker_board,
+)
+from ...core import ResolveInfo
+from ...core.doc_category import DOC_CATEGORY_APPS
+from ...core.mutations import BaseMutation
+from ...core.types import AppError
+from ...utils import get_user_or_app_from_context
+
+logger = logging.getLogger(__name__)
+
+breaker_board = initialize_breaker_board()
+
+
+class AppReenableSyncWebhooks(BaseMutation):
+    app = graphene.Field(
+        App,
+        description="App for which sync webhooks were re-enabled.",
+    )
+
+    class Arguments:
+        app_id = graphene.ID(
+            description="The app ID to re-enable sync webhooks for.", required=True
+        )
+
+    class Meta:
+        description = (
+            "Re-enable sync webhooks for provided app. "
+            "Can be used to manually re-enable sync webhooks for the app before "
+            "the cooldown period ends." + ADDED_IN_321
+        )
+        doc_category = DOC_CATEGORY_APPS
+        permissions = (AppPermission.MANAGE_APPS,)
```

Comment:
Not sure if this is a valid scenario, but do we plan to allow the app, to re-enable its owns webhooks?